javatests/arcs/core/storage/driver/DatabaseDriverTest.kt

```diff
@@ -364,29 +364,23 @@ class DatabaseDriverTest {
     assertThat(receiverVersion).isEqualTo(1)
   }
 
-  @Test
-  fun send_throws_ifUnsupportedDataType() = runBlockingTest {
-    val driver = buildDriver<Int>(database)
-
-    assertFailsWith(
-      UnsupportedOperationException::class,
-      "Unsupported type for DatabaseDriver: ${Int::class}"
-    ) {
-      driver.send(12, 1)
-    }
-  }
```

Comment:
any reason for removing this?

Reply:
I removed it last week, so my memory is fuzzy. However, I believe that when refactoring this test, I found that there was no way to structure it to have incorrect types, due to the generic type constraints. So, I removed the test.